elements/rh-secondary-nav/rh-secondary-nav.css

```diff
@@ -0,0 +1,147 @@
+:host {
+  position: sticky;
+  top: 0;
+  z-index: var(--rh-secondary-nav-z-index);
+  font-size: initial;
+}
+
+a {
+  text-decoration: none;
+  color: var(--rh-secondary-nav-link-color);
+}
+
+nav {
+  position: relative;
+  height: 100%;
+  z-index: var(--rh-secondary-nav-z-index);
+}
+
+#container {
+  display: grid;
+  position: relative;
+  grid-template-areas: 'logo menu' 'nav nav' 'cta cta';
+  grid-template-columns: 1fr max-content;
+  grid-template-rows: var(--rh-secondary-nav-min-height) max-content max-content;
+  column-gap: 1em;
+  row-gap: 0;
+  height: fit-content;
+  background-color: var(--rh-secondary-nav-background-color);
+  z-index: var(--rh-secondary-nav-z-index);
+}
+
+#container.expanded ::slotted([slot="nav"]) {
+  display: flex !important;
+  list-style: none;
+  flex-direction: column;
+  padding: 2em 1em 1em;
+  margin: 0 !important;
+  max-height: calc(100vh - var(--rh-secondary-nav-min-height ,4.625em));
+  overflow-y: auto;
+}
+
+nav.is-mobile #container.expanded ::slotted([slot="nav"]) {
+  background-color: var(--rh-secondary-nav-expanded-background-color);
+}
+
+#container.expanded ::slotted([slot="cta"]) {
+  display: flex !important;
+  padding: 1em 1em 2em;
+}
+
+nav.is-mobile #container.expanded #cta {
+  background-color: var(--rh-secondary-nav-expanded-background-color);
+}
+
+button {
+  grid-area: menu;
+  border: none;
+  display: flex;
+  height: 100%;
+  align-items: center;
+  font-family: var(--pfe-theme--font-family);
+  font-size: 1em;
+  padding: 1em;
+  border-top: 0.25em solid transparent;
+  margin-right: 1em;
+  color: var(--rh-secondary-nav-button-color);
+  background-color: var(--rh-secondary-nav-background-color);
+}
+
+button:hover {
+  color: var(--rh-secondary-nav-button-hover-color);
+  border-top-color: var(--rh-secondary-nav-brand-color);
+}
+
+
+button:after {
```

Comment:
```suggestion
button::after {
```

Reply:
Fixed in 190153fba40663184149e3b5a41720e7b49f1dd6